app/main/views/suppliers.py

```diff
@@ -64,11 +66,18 @@ def list_suppliers():
     else:
         suppliers = Supplier.query.order_by(Supplier.name, Supplier.supplier_id)
 
+    # Can search by either DUNS or Company Registration number but not both
```

Comment:
Should we raise an error if the user tries to search using both?

Reply:
I've just ignored it for now, the apps aren't ever likely to search by both at once. I think in general we ignore 'bad' query strings rather than raising errors. Happy to change if people feel differently?